adding_species_manuscript.tex

```diff
@@ -652,29 +656,13 @@ \subsection*{Filling out the missing pieces}
 \end{table}
 
 For many species it is difficult to obtain estimates of all necessary model parameters.
-Table \ref{tab:param-mod} provide suggestions for ways to deal with missing values of various central model parameters.
-The table also mentions the main discrepancies between the simulated data and real genomic data,
-which can be caused by misspecification of each parameter.
-
-Several researchers who participated in the ``Growing the Zoo'' hackathon wished to add species
-whose genome assemblies are composed of many relatively small contigs,
-unanchored to chromosome-level scaffolds.
-Although we had not previously put restrictions on which species might be added,
-we decided that we would only add species with chromosome-level assemblies.
-The main justification for this restriction is that
-species with less complete genome builds typically do not have good estimates of recombination rate, genetic maps, and demographic models,
-making chromosome-level simulation much less useful in such species.
-Another issue is the storage burden and long load times involved in dealing with
-hundreds of contigs.
-Finally, each species requires validation of its code before it is added to the \stdpopsim catalog,
-as well as long-term maintenance to keep it up-to-date with changes made to the \stdpopsim framework.
-So, the benefit of including species with very partial genome builds in \stdpopsim
-would be outweighed by the substantial extra burden on \stdpopsim maintainers as well as
-downstream users of these models.
-
-
-%Ilan: applied some important edits in this paragraph to better address some of the delicate situations and options we suggest for them 
-That being said, simulation is still potentially useful for species with partial genome builds.
+Table \ref{tab:param-mod} provides suggestions for ways to deal with missing values of various central model parameters.
+The table also mentions possible consequences of misspecification of each parameter.
+
+
+There are additional considerations that apply to species with partial genome builds.
+We discussed above our choice in \stdpopsim to focus on species with chromosome-level assemblies.
+However, simulation is still potentially useful for species with partial genome builds.
```

Comment:
How about deleting the first sentence, "There are additional considerations that apply to species with partial genome builds.", and adding to the end of this sentence ", with some additional considerations".  (note the comma at the start of the added text)

Reply:
Hmm, feels like it would be OK to me; since it's a new paragraph, a shift in topic is expected.  But any edit is fine, as long as it gets rid of the large amount of redundancy between the first and third sentences above.  Maybe you can consolidate them into the first sentence, instead of consolidating them into the third sentence as I suggested.

Reply:
I made another edit trying to fix this. See below